scripts/integration_test.sh

```diff
@@ -28,6 +28,10 @@ test -d ${CACHE_DIR:-cache}/mozilla-central
 ls -lh
 ls -lh data
 
+# Then generate a sample fixed comment dataset
+bugbug-fixed-comments
+ls -lh
+ls -lh data
```

Comment:
We should not add it to the integration test without a limit, so we may want to do it in a separate PR later. Could you please file an issue for this?

Reply:
Done here: https://github.com/mozilla/bugbug/issues/4466

Reply:
Removed the integration test here: 2ac50b95598cddac70d59015c4d41e2df3936b12